### ISSUE TITLE
Information: Smart Battery Protect

### ISSUE THREAD

**Issue by spattinson** [Fabian-Schmidt/esphome-victron_ble#68]
It works fine. The text sensors do not output anything under normal conditions
I tested turning the output off using the app
device_state 249 to 0
off_reason 0 to 4
output_state 1 to 0 - I guess this means whether the device is on or off, through manual switching or alarm
output_voltage changed from 13.6V to 0.4V
I did not configure battery voltage, but it should be the same as input_voltage
I have not configured any binary sensors.
alarm_reason, error, warning_reason are all zero as expected - I could raise the threshold voltage in the app and wait for the voltage to go below that to get different values if required?
Thanks for providing this.

**Comment by Fabian-Schmidt** [comment]
Thanks you.

I added the device state 249 in #81 . The text value is `Load detect`.

I also added the output_state as a text sensor using the values you provided.

I would expect the text sensor for off_reason to output something given the values you provided.

Thank you for your input.

**Comment by VasilisMavr** [comment]
Hi, I have a question, request if possible?

In victron app for the Smart Battery Protect you can trigger the device to isolate the battery or rengage it.

Is it possible with this intergration to add a switch to do this?

Thanks

vas

**Comment by spattinson** [comment]
> Hi, I have a question, request if possible?
> 
> In victron app for the Smart Battery Protect you can trigger the device to isolate the battery or rengage it.
> 
> Is it possible with this intergration to add a switch to do this?
> 
> Thanks
> 
> vas

I dont think so this listens to ble broadcasts, it transmits nothing and is not paired
